app/views/catalog/_bulk_index_table.html.erb

```diff
@@ -1,7 +1,7 @@
 <div id="bulk-upload-table">
     <table class="table table-hover">
 	<tr>
-	    <th>When</th><th>Who</th><th>File Name</th><th>Note</th><th>Status <%= link_to bulk_status_help_path(@document.get('id')), :data => { ajax_modal: 'trigger', ajax_modal_title: 'Status Information' } do %><span class="glyphicon glyphicon-question-sign" aria-hidden="true"></span><% end %></th><th>Druids</th><th>Success</th><th/><th/>
```

Comment:
Is it possible to pass just `@document` to these path helpers, or do the route not line up quite right?